optimize/precondition.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from ff.handlers.bonded import HarmonicAngleHandler, HarmonicBondHandler, ImproperTorsionHandler, ProperTorsionHandler
+from ff.handlers.nonbonded import LennardJonesHandler, AM1CCCHandler
+
+default_learning_rates = {
+    HarmonicBondHandler:    np.zeros(2),    # k, length
+    HarmonicAngleHandler:   np.zeros(2),    # k, angle
+    ProperTorsionHandler:   np.zeros(3),    # k, phase, periodicity
+    ImproperTorsionHandler: np.zeros(3),    # k, phase, periodicity
+    AM1CCCHandler:          np.ones(1),     # charge increment
+    LennardJonesHandler:  np.array([0, 1]), # sigma, epsilon
```

Comment:
Intentionally disabling sigma? For some reason I thought it should be the other way around.... (disable epsilon but not sigma).

Reply:
Unintentional, had things backwards. Addressed in https://github.com/proteneer/timemachine/pull/490/commits/c01ae0a5f901bc9a012e851f381a056e29a35ed8